client.go

```diff
@@ -146,6 +155,39 @@ func (client Client) NewReq(method, uri string, body io.Reader, mods ...func(*Re
 	return req
 }
 
+func (client Client) NewRawReq(method, uri string, body io.Reader, mods ...func(*Req)) Req {
+	httpReq, _ := http.NewRequest(method, client.Url+uri, body)
+	req := Req{
+		HttpReq:    httpReq,
+		LogPayload: false,
+	}
+	for _, mod := range mods {
+		mod(&req)
+	}
+	return req
+}
+
+func (client Client) NewReqWithQueryString(method, uri string, queryString []string, mods ...func(*Req)) Req {
```

Comment:
Instead of duplicating the code again, I would suggest implementing this as a request option. See:

https://github.com/netascode/go-aci/blob/main/README.md#query-parameters

https://github.com/netascode/go-aci/blob/5f439c8ed9e1209f4e6877c55303f977aea9c635/req.go#L79